docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,112 @@
+
+- Using ConfigMaps
+
+    Store and manage configuration data as key-value pairs. ConfigMaps are frequently used to tune platform-wide behavior. Most of the Knative ConfigMaps are in the `knative-serving` and `knative-eventing` namespaces. Their settings apply to all the relevant Knative components in all namespaces.
+
+- Applying resources with the `kubectl` CLI
+
+    Apply updated YAML manifests, as opposed to ConfigMaps, to the cluster using standard Kubernetes workflows.
+
+- Using the Knative Operator
+
+    Some platform-wide settings can be managed declaratively using the Knative Operator.
+
+### Configuration tasks
+
+Knative documentation provides the following configuration procedures. This list is subject subject to change.
+
+Configurations for default settings:
+
+- [Broker defaults](../eventing/configuration/broker-configuration.md)
+- [ConfigMap defaults](../serving/configuration/config-defaults.md)
+- [Event source defaults](../eventing/configuration/sources-configuration.md)
+- [Channel defaults](../eventing/configuration/channel-configuration.md)
+- [Kafka channel defaults](../eventing/configuration/kafka-channel-configuration.md)
+- [Domain names](../serving/using-a-custom-domain.md)
+- [Ingress gateway replacement](../serving/setting-up-custom-ingress-gateway.md)
+
+Configurations for new development:
+
+- [Deployment resources](../serving/configuration/deployment.md)
+- [Istio access to deployed services](../serving/istio-authorization.md)
+- [Namespace exclusion from webhooks](../serving/istio-authorization.md)
+
+Configurations for maintenance:
+
+- [Garbage collection](../serving/revisions/revision-admin-config-options.md)
+- [High availability](../serving/config-ha.md)
+- [Rollout duration for revisions](../serving/configuration/rolling-out-latest-revision-configmap.md)
+- [Autoscaling of Kafka features](../eventing/configuration/keda-configuration.md)
+
+Configurations for security encryptions:
+
+- [cert-manager](../serving/encryption/configure-certmanager-integration.md)
+- [External domains](../serving/encryption/external-domain-tls.md)
+- [Local domains](../serving/encryption/cluster-local-domain-tls.md)
+- [system-internal](../serving/encryption/system-internal-tls.md)
+
+Configurations for extensions:
+
+- [Kafka Broker features](../serving/encryption/system-internal-tls.md)
+- [Sugar Controller](../eventing/configuration/sugar-configuration.md)
+
+Configurations for flagging features:
+
+- [Serving features](../serving/configuration/feature-flags.md)
+- [Eventing features](../eventing/features/README.md)
+
+## Authorizations
+
+You can grant developers access to additional resources related to their namespace in other services, such as observability, logs, metrics, tracing, and dashboards.
```

Comment:
Not for this PR, but we should expand this to also talk about how application access can be managed by using ingress and Eventing features to require specific types of tokens (authentication and authorization) to deliver requests or events.